app/models/course.rb

```diff
@@ -222,6 +224,7 @@ def self.ready_for_survey(opts)
   before_save :check_course_times
   before_save :set_needs_update
   after_create :ensure_home_wiki_in_courses_wikis
+  after_create :create_statistic
```

Comment:
I don't think this is necessary. You'll need to account for already-existing courses that won't have a statistics record, anyway.